src/blueapi/config.py

```diff
@@ -85,6 +85,16 @@ class RestConfig(BlueapiBaseModel):
     protocol: str = "http"
 
 
+class ScratchRepository(BlueapiBaseModel):
```

Comment:
Eventually we may want to specify a branch too?

I suppose at this stage people will still be editing the scratch directory directly for modifications so it's probably good that the init doesn't mess with the working tree if the repo is already present.

Reply:
I went back and forth on this, yeah, not even sure if we want a branch. If we're aiming for a world of automatic commits, a linear history is the easiest thing to manage...

Reply:
As discussed this is sufficient for now.